templates/python/custom-copilot-basic/teamsapp.local.yml.tpl

```diff
@@ -0,0 +1,81 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/1.0.0/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: 1.0.0
```

Comment:
use `v1.3` to bring latest feature to users. can discuss whether we want to use `v1.4` which will be released soon.